homeassistant/components/sonos/media_player.py

```diff
@@ -511,16 +516,16 @@ def _set_favorites(self):
 
     def _radio_artwork(self, url):
         """Return the private URL with artwork for a radio stream."""
-        if url not in ("", "NOT_IMPLEMENTED", None):
-            if url.find("tts_proxy") > 0:
-                # If the content is a tts don't try to fetch an image from it.
-                return None
-            url = "http://{host}:{port}/getaa?s=1&u={uri}".format(
-                host=self.soco.ip_address,
-                port=1400,
-                uri=urllib.parse.quote(url, safe=""),
-            )
-        return url
+        if url in UNAVAILABLE_VALUES:
+            return None
+
+        if url.find("tts_proxy") > 0:
+            # If the content is a tts don't try to fetch an image from it.
+            return None
+
+        return "http://{host}:{port}/getaa?s=1&u={uri}".format(
```

Comment:
Normally we would want to have this in the device library.

Reply:
`f""`

Reply:
I agree but we don't have a maintainer for this integration. This is existing code that got its indentation changed. Converted it now to an f-string.

Reply:
I never got around to this part. I can submit a separate PR with a cleanup.